pkg/utils/slice/slice.go

```diff
@@ -14,8 +14,8 @@
 
 package slice
 
-// ContainsString checks if a given slice of strings contains the provided string.
-func ContainsString(slice []string, s string) bool {
+// Contains checks if a given slice contains the provided element.
+func Contains[K comparable](slice []K, s K) bool {
```

Comment:
I think we could delete this function and use the new Contains() from standard library package slices